### PR TITLE
docs: rewrite and consolidate docs about backups

### PR DESCRIPTION
**Pull request by rocketeerbkw**
# General Checklist

- [x] Affected Issues have been mentioned in the Closing issues section
- [x] Documentation has been written/updated
- [x] PR title is ready for inclusion in changelog

Rewrite of the backup docs to clear up some info for https://github.com/uselagoon/build-deploy-tool/pull/372 and to document previously undisclosed environment variables.

I moved info from other pages (environment variables and FAQ) that were backup related. Not sure if this is a great idea, I'm open to discussion.

Some changes were easy to make in the japanese language pages, but the big rewrite will require more work than I can do.

### CODE REVIEW

**Comment by shreddedbacon** [comment]
Quick skim this looks good. Consolidating all the information down is a great idea. Will have a better run over it when I get a bit more time.

File: docs/concepts-advanced/backups.md [hunk 1]
```diff
@@ -1,37 +1,222 @@
...
+For increased security and reduced storage costs you can opt into [removing
+restored backups after a set
+lifetime](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lifecycle-mgmt.html)
+(e.g. 7 days). Lagoon caters for this scenario gracefully and will re-create any
+restored snapshots as needed.
+
+## Manual Backups
+
+Users can create manual backups by running a backup task, syncing data from one
+environment to another, or by running custom commands in containers.
```
Comment (shreddedbacon) [inline docs/concepts-advanced/backups.md]: I would avoid referring to the task as a `backup` task. Not sure how else to describe it as more than a database dump task?

Comment (shreddedbacon) [reply docs/concepts-advanced/backups.md]: Otherwise we don't have a "manual backup" button (yet?)

Comment (rocketeerbkw) [reply docs/concepts-advanced/backups.md]: Ya, that's probably more clear for current state: manual backups aren't "created," they're done/run/performed. So something like:

> Users can manually backup databases and files by running a task, syncing.....

If we add the ability to do on-demand backups from the backups tab, that can be added to the list of manual backup methods.

Comment (shreddedbacon) [reply docs/concepts-advanced/backups.md]: Yah sounds good

**Review by shreddedbacon — APPROVED** [review]
Yep, makes sense.

Unsure about the :jp: translations though

**Pull request merged**